shark_turbine/kernel/wave/expansion.py

```diff
@@ -261,6 +261,9 @@ def _expand_node(
     new_node.fx_node.name = get_expanded_name(node, restricted_dims)
     node_index_setter(new_node, restricted_dims)
 
+    constraints = node_index_setter.args[0]
+    new_node.pre_expansion(constraints)
```

Comment:
I don't think we are using pre_expansion for anything right now? Is there a plan to do so or is it just for symmetry? haha

Reply:
Yes, not being used right now. I can remove until we need it.